gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicationReplica.java

```diff
@@ -0,0 +1,16 @@
+package gobblin.data.management.copy.replication;
+
+import gobblin.data.management.copy.CopyableDataset;
+import gobblin.dataset.DatasetsFinder;
+
+public class ReplicationReplica extends AbstractReplicationData {
```

Comment:
I used ReplicationSource and ReplicationReplica in pair, if change to Replica, change ReplicationSource to Source ? I think that is too big a name for "source"